guides/source/testing.md

```diff
@@ -1671,7 +1671,7 @@ which can be used in system tests.
 | `assert_current_path(string, **options)`                         | Asserts that the page has the given path. |
 | `assert_field(locator = nil, **options, &optional_filter_block)` | Checks if the page has a form field with the given label, name or id. |
 | `assert_link(locator = nil, **options, &optional_filter_block)`  | Checks if the page has a link with the given text or id. |
-| `assert_selector(*args, &optional_filter_block)`                 | Asserts that a given selector is on the page. |
+| `assert_select(*args, &optional_filter_block)`                   | Asserts that a given selector is on the page. |
```

Comment:
```suggestion
| `assert_selector(*args, &optional_filter_block)`                   | Asserts that a given selector is on the page. |
```

Reply:
@p8 unfortunately, these two assertions are not equivalent. Capybara's [assert_selector](https://rubydoc.info/github/teamcapybara/capybara/master/Capybara/Selector) works for both string (treated as CSS or XPath) and symbol (Capybara built-in selector values like `:field`, `:button`, `:link`).

The [assert_select](https://rubydoc.info/github/teamcapybara/capybara/master/Capybara/Node/Matchers#has_select%3F-instance_method) Capybara helper is specific to `<select>` elements.